docs/src/What's New.doc.js

```diff
@@ -0,0 +1,74 @@
+// @flow strict
+import React, { type Node, useEffect } from 'react';
+import { Box, Text, Link, Stack, Row } from 'gestalt';
+import Markdown from './components/Markdown.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+function Changelog() {
+  const [changelogData, setChangelogData] = React.useState('Loading...');
+
+  const getChangelog = () => {
+    fetch('https://cdn.jsdelivr.net/gh/pinterest/gestalt@master/CHANGELOG.md')
+      .then(response => {
+        if (!response.ok) {
+          return '# There was error loading the changelog, please try again later.';
```

Comment:
@ayeshakmaz looks like this does not work as expected, it keep showing 'Loading...' for me.

You can try it out by:
1) Loading the `What's new page`
2) Going to a different page
3) Disconnect from the internet
4) Go back to the `What's new page`
![image](https://user-images.githubusercontent.com/127199/96319863-a5d36280-0fc5-11eb-81cb-a43fe422d5df.png)

Reply:
Ahh this is because the default value for the changelog data is `Loading...` 

If you disconnect from the internet, it should serve from cache, but if I'm mistaken that fetch wouldn't happen at all in offline mode right?

Reply:
Afaik the fetch should happen but might fail - one thing we can try is to wrap it in a `try/catch`